apitools/base/py/list_pager.py

```diff
@@ -94,6 +95,7 @@ def YieldFromList(
           response message holding the maximum number of results to be
           returned. None if caller-specified batch size is unsupported.
           If a tuple, path to the attribute.
+      get_field_fn: lambda, A function that returns the items to be yielded.
```

Comment:
more consistent to use
```
get_field_func: Function that...
```

https://github.com/google/apitools/blob/master/apitools/base/py/http_wrapper.py#L328